editoast/Cargo.toml

```diff
@@ -199,6 +200,7 @@ axum = { version = "0.7.9", default-features = false, features = [
 editoast_authz = { workspace = true, features = ["fixtures"] }
 editoast_models = { workspace = true, features = ["testing"] }
 editoast_osrdyne_client = { workspace = true, features = ["mock_client"] }
+opentelemetry_sdk = { workspace = true, features = ["testing"] }
```

Comment:
Feature `testing` requires `async-std`, which we don't want since we use `tokio`. It's already in our dev-dependencies because `rstest` requires it, but we'll be able to lift that restriction when we'll have our `DbPoolV3`. In the meantime, could we remove this feature and implement a `NoopExporter` manually?

Reply:
Done.